src-tauri/src/analytics/algo_result_enums/diff_cen.rs

```diff
@@ -2,7 +2,7 @@ use std::collections::HashMap;
 
 #[derive(Debug)]
 pub enum DiffCenResult {
-    Success(HashMap<String, HashMap<String, f64>>),
+    Success(HashMap<String, HashMap<String, HashMap<String, f64>>>),
```

Comment:
Why has this changed from a hashmap of hashmaps to a hashmap of hashmap of hashmaps? ...This is the best sentence I've ever typed in my life.

Reply:
If I'm understanding correctly, each timestep maps to a full set of nodes (that's our first hashmap). Each of those nodes maps to a bunch of floats that's a diffcen result (that's our second hashmap).

Reply:
Anyways you'll get serialization when you pull from my PR (merged), which may fix some of the errors.

Reply:
Yeah we are returning all timestep results now for the chart so that user can see how diffusion changes over time